go.mod

```diff
@@ -619,7 +617,11 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
-require github.com/pyroscope-io/godeltaprof v0.1.1
+require (
```

Comment:
Do you mind placing this in the first `require` block in the go.mod please? (the "direct" one)

Reply:
sure